st2client/st2client/commands/keyvalue.py

```diff
@@ -168,6 +168,10 @@ def __init__(self, resource, *args, **kwargs):
         self.parser.add_argument('-e', '--encrypt', dest='secret',
                                  action='store_true',
                                  help='Encrypt value before saving.')
+        self.parser.add_argument('--pre-encrypted', dest='pre_encrypted',
+                                 action='store_true',
+                                 help=('Value provided is already encrypted with the instance '
+                                 'crypto key and should be stored as-is.'))
```

Comment:
Can we make the args `--encrypt` and `--encrypted/--pre-encrypted` mutually exclusive?  Meaning these args cannot both be present at the same time.

Reply:
Good point, will add it.